### PR TITLE
[v17] Update e ref

### PR DESCRIPTION
**Pull request by smallinsky**

### CODE REVIEW

**Comment by marcoandredinis** [comment]
This seems to bring commits from `/e/` `master`. Is it expected?

File: e
Comment (codingllama) [inline e:1]: This is pulling from e/ master, not e/ branch/v17.

**Review by marcoandredinis — APPROVED** [review]

**Comment by smallinsky** [comment]
> This seems to bring commits from /e/ master. Is it expected?

Not this is not intended. Fixed. Thanks for catching this.

**Review by codingllama — APPROVED** [review]
Thanks!

**Pull request merged**